cmd/build.go

```diff
@@ -37,15 +47,17 @@ type Registry struct {
 	Url             string
 	Username_Envvar string
 	Password_Envvar string
-	username        string `default:""`
-	password        string `default:""`
+	Username        string `default:""`
+	Password        string `default:""`
 }
 
-type Image struct {
-	name    string
-	context string
+type Verbose struct {
+	msg          string
+	return_value string
 }
 
+type ExternalProgramOnFile func(string, *bytes.Buffer) error
```

Comment:
tbh, I was not really sure what to name it. I wanted a type for the code style program that checks python source files, so that I could inject a different dependency in the unit tests.